azurepipelines/e2e_test/templates/e2e_test_run.yaml

```diff
@@ -7,6 +7,12 @@ parameters:
       type: string
 
 steps:
+    - task: PipAuthenticate@1
+      displayName: 'Pip Authenticate'
+      inputs:
+          artifactFeeds: 'adu-linux-client/ADUTestFeed'
+          onlyExtraIndex: true
```

Comment:
Why not making it the primary index URL?  I see `onlyAddExtraIndex` but not `onlyExtraIndex` in the pip authenticate v1 doc: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/pip-authenticate-v1?view=azure-pipelines